### PR TITLE
Try to fix linter Error

### PR DESCRIPTION
**Pull request by yohannesdagnachew**
This will help to fix  linter error

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -43,20 +43,6 @@ jobs:
...
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-      - name: Setup ESLint
-        run: |
-          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
-          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.eslintrc.json
-      - name: ESLint Report
-        run: npx eslint .
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (brytebee) [inline .github/workflows/linters.yml:48]: - [ ] **Required:** You are not permitted to edit the linters file. You need the **Esliint** checks for your JS code.

**I have dropped a couple of messages on Slack to see if could walk you through the linters set up but you did not reply.**

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @yohannesdagnachew👋, 

Great work on making the changes 👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by yohannesdagnachew** [comment]
I don't have .js file in my project . I don't edit the linters file I use
wrong linter file first

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (ShahierNashaat) [inline .gitignore:1]: - [ ] [Optional] Please add node_modules to your .gitignore file to make more organized.

**Review by ShahierNashaat — APPROVED** [review]
Hi @yohannesdagnachew , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions
- [ ] Please add more descriptions to your pull request.
- [ ] Please make the title of your pull request more descriptive
- [ ] Please choose a descriptive name to your branch.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**